apps/web/src/components/media-gallery/media-gallery.tsx

```diff
@@ -102,11 +102,12 @@ export default function MediaGallery({ media }: MediaGalleryProps) {
                 onClick={() => handleMediaChange(media)}
               >
                 <Image
-                  alt=""
+                  alt={t('common:actions.View image X of Y')}
```

Comment:
Using a template translation (grab `i` from the map call):

```suggestion
                  alt={t('common:actions.View image X of Y', { x: i, y: media.length )}
```